search/lib/validator/stacValidator.js

```diff
@@ -0,0 +1,19 @@
+function createStacValidator (stacItem) {
+  if (!stacItem.stac_version || !stacItem.id || !stacItem.description || !stacItem.links) throw Error('Missing required fields');
+}
+
+function validateStac (stacItem) {
+  if (!stacItem) throw new Error('Missing stacItem');
+  try {
+    createStacValidator(stacItem);
+  } catch (e) {
+    return false;
+  }
+
+  return true;
+}
```

Comment:
I feel like these two functions could be combined into a single `validateStac()` function. The current `validateStac()` isn't really doing anything except calling `createStacValidator()`.